src/components/Mission.jsx

```diff
@@ -0,0 +1,60 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { displayMissions } from '../Redux/Missions/missionSlice';
+import Specificmission from './Specificmission';
+
+const Mission = () => {
+  const { missions } = useSelector((state) => state.missions);
+
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (missions.length === 0) {
+      dispatch(displayMissions());
+    }
+  }, [dispatch, missions.length]);
+
+  return (
+    <div className="max-w-6xl mx-auto">
+      <table className="table-fixed border dark:border-neutral-500 mt-4 mb-4">
+        <thead className="border-b font-medium dark:border-neutral-500">
+          <tr>
+            <th
+              scope="col"
+              className="border-r px-6 py-4 dark:border-neutral-500"
+            >
+              Mission
+            </th>
+            <th
+              scope="col"
+              className="border-r px-6 py-4 dark:border-neutral-500"
+            >
+              Description
+            </th>
+            <th
+              scope="col"
+              className="border-r px-6 py-4 dark:border-neutral-500"
+            >
+              Status
+            </th>
+            <th
+              scope="col"
+              className="border-r px-6 py-4 dark:border-neutral-500"
+            >
+              &lsquo;
+            </th>
+          </tr>
```

Comment:
- [ ] [OPTIONAL] For some reasons yet unknown but I suspect it must be because of the available space, one of your missions is not displaying the actionable buttons in case users want to indicate interest in the mission by joining. It would be nice to work on that even if the change is optional. :clap: 

![image](https://github.com/Kanyorok/spacex-project/assets/99715640/98e18d9a-6fb4-4a2f-a59f-a3819eb6113d)